packages/builders/__tests__/components/components.test.ts

```diff
@@ -1,21 +1,19 @@
-import {
+import type {
 	APIActionRowComponent,
 	APIButtonComponent,
 	APIMessageActionRowComponent,
 	APISelectMenuComponent,
 	APITextInputComponent,
-	ButtonStyle,
-	ComponentType,
-	TextInputStyle,
 } from 'discord-api-types/v10';
+import { ButtonStyle, ComponentType, TextInputStyle } from 'discord-api-types/v10';
```

Comment:
use `type` in a single import. Same for other files

Reply:
That's not allowed with this eslint config

Reply:
If I'm not wrong, https://github.com/iCrawl/eslint-config-neon/blob/main/src/typescript.js#L48 rule allows it but if there is a type import without `type`, the auto fix makes it two line. So if u manually change all the imports to use
```ts
import { name, type name } from 'name';
```
it'll be valid.

Reply:
That is true.